modules/installation-special-config-storage.adoc

```diff
@@ -127,44 +170,100 @@ boot_device:
   luks: <3>
     tpm2: true <4>
     tang: <5>
-      - url: https://tang.example.com
-        thumbprint: eyJhbmc3SlRyMXpPenc3ajhEQ01tZVJiTi1oM...
-    threshold: 1 <6>
-  mirror: <7>
-    devices: <8>
+      - url: http://tang.example.com:7500 <6>
+        thumbprint: PLjNyRdGw03zlRoGjQYMahSZGu9 <7>
+    threshold: 1 <8>
+  mirror: <9>
+    devices: <10>
       - /dev/sda
       - /dev/sdb
 openshift:
-  fips: true <9>
+  fips: true <11>
 ----
 +
-<1> On control plane nodes, substitute `master` for `worker` in both of these locations.
+<1> For control plane configurations, replace `worker` with `master` in both of these locations.
 <2> On ppc64le nodes, set this field to `ppc64le`.  On all other nodes, this field can be omitted.
-<3> Include this section if you want to encrypt the root filesystem. See "About disk encryption" for more details.
+<3> Include this section if you want to encrypt the root filesystem. See the _About disk encryption_ section for more details.
 <4> Include this field if you want to use a Trusted Platform Module (TPM) to encrypt the root filesystem.
-<5> Include this section if you want to use a Tang server. To obtain the server URL and thumbprint, follow the preceding instructions.
-<6> Include this directive to use Shamir's Secret Sharing (SSS) to require the specified number of Tang servers, or one less than the specified number of servers plus a TPM, to unlock the disk.
-<7> Include this section if you want to mirror the boot disk. See "About disk mirroring" for more details.
-<8> List all disk devices that should be included in the boot disk mirror, including the disk that {op-system} will be installed onto.
-<9> Include this directive to enable FIPS mode on your cluster.
+<5> Include this section if you want to use one or more Tang servers.
+<6> Specify the URL of a Tang server. In this example, `tangd.socket` is listening on port `7500` on the Tang server.
+<7> Specify the exchange key thumbprint, which was generated in a preceding step.
+<8> Specify the minimum number of TPM v2 and Tang encryption conditions that must be met for decryption to occur. The default value is `1`. See the _Configuring an encryption threshold_ section for more information on this topic.
```

Comment:
If you changed to For more information, see above, same here.

Reply:
Resolved.